extra-i18n/wps-office-spellcheck/autobuild/build

```diff
@@ -0,0 +1,23 @@
+mkdir -p ${PKGDIR}/usr/lib/office6/data/
+wget -c http://wps-community.org/download/dicts/ca_ES.zip
```

Comment:
Never considered a for loop? Btw wget supports multiple URLs, so you can write `wget -c http://wps-community.org/download/dicts/{ca_ES,cs_CZ,blah}.zip`.